### ISSUE TITLE
REACH tests failing on Travis

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#290]
Java appears to be running out of heap space on the first test, causing all following tests to fail.

From Travis build 811.1:

```======================================================================

ERROR: test_reach.test_phosphorylate

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/sorgerlab/indra/indra/tests/test_reach.py", line 70, in test_phosphorylate

    assert(len(rp.statements) == 1)

AttributeError: 'NoneType' object has no attribute 'statements'

-------------------- >> begin captured logging << --------------------

reach: ERROR: Could not process text.

reach: ERROR: JVM exception occurred: Java heap space

--------------------- >> end captured logging << ---------------------

======================================================================

ERROR: test_reach.test_regulate_amount

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/sorgerlab/indra/indra/tests/test_reach.py", line 80, in test_regulate_amount

    assert(len(rp.statements) == 1)

AttributeError: 'NoneType' object has no attribute 'statements'

-------------------- >> begin captured logging << --------------------

reach: ERROR: Could not process text.

reach: ERROR: JVM exception occurred: Could not initialize class org.clulab.reach.export.apis.ApiRuler$

--------------------- >> end captured logging << ---------------------
```

**Comment by bgyori** [comment]
This is due to: https://docs.travis-ci.com/user/build-environment-updates/2017-09-06/
```
Added definition of _JAVA_OPTIONS="-Xmx2048m -Xms512m"
```

**Comment by bgyori** [comment]
Okay, fixed in 3c27f59aad61da9110be5e29a4aca43fcdbd2200. Just need to explicitly set _JAVA_OPTIONS before running nosetests.
